doc/api.yml

```diff
@@ -1181,6 +1181,115 @@ paths:
           $ref: '#/components/responses/429'
         '500':
           $ref: '#/components/responses/500'
+  '/students/{id}/pointing':
+    post:
+      tags:
+        - Check-in
+      summary: To monitoring the pointing of one student
+      operationId: createPointing
+      parameters:
+        - name: id
+          in: path
+          required: true
+          description: give the student id
+          schema:
+            type: string
+        - name: place
+          in: query
+          required: true
+          description: give the place where the pointing was registered
+          schema:
+            type: string
+        - name: pointing_type
+          in: query
+          required: true
+          schema:
+            $ref: '#/components/schemas/PointingType'
+      responses:
+        '200':
+          description: monitoring a student pointing
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Pointing'
+        '400':
+          $ref: '#/components/responses/400'
+        '403':
+          $ref: '#/components/responses/403'
+        '404':
+          $ref: '#/components/responses/404'
+        '429':
+          $ref: '#/components/responses/429'
+        '500':
+          $ref: '#/components/responses/500'
+  '/courses/{course_id}/pointing':
+    get:
+      tags:
+        - Check-in
+      summary: Get all student who missing a course or are late
```

Comment:
Why endpoint courses returns students ? Must be from students and just parameterized by a course query params.